### DISCUSSION TITLE
Security Audit Does Not Appear in Landscape

### DISCUSSION THREAD

**Discussion by rockbison13** [cncf/landscape2#762] [Q&A] [answered]
Hi. I'm currently trying to create my company landscape, however I'm having some problem to produce the security audit box in picture [1]. Would really appreciate if anyone in this community can give me a pointer to solve this. Thank you :)
![Screenshot 2024-12-04 174351](https://github.com/user-attachments/assets/6491ce40-e1b4-46ee-b349-be7d508035eb)

![Screenshot 2024-12-04 172631](https://github.com/user-attachments/assets/24e828ff-16ea-4abf-8a35-e6c063351070)
![Screenshot 2024-11-27 165829](https://github.com/user-attachments/assets/9aa3d43e-eef4-4bc9-9a97-d57045f611f9)

**Comment by tegioz** [comment, marked as answer]
Hi @rockbison13 👋 

They need to be included in the `audits` field, inside `extra`:

<https://github.com/cncf/landscape2/blob/896bbf6e3757c5258f408c39d0acb18d58ba236f/docs/config/data.yml#L116-L128>

**Reply by rockbison13** [reply]
Hi. Thank you for the reply. I have included the audits part as shown in Picture [2]. However, it still didn't show up. Is it because of the spacing or something else?

**Reply by rockbison13** [reply]
I've solved it. My problem is with the spacing. Thank you @tegioz! :)
![image](https://github.com/user-attachments/assets/77f2b0b0-1e8b-4649-9ec8-af6f74177484)

**Reply by tegioz** [reply]
Awesome, no problem 👍
